xlators/mgmt/glusterd/src/glusterd-svc-helper.c

```diff
@@ -360,7 +353,7 @@ glusterd_volume_svc_check_topology_identical(
     int tmpclean = 0;
     int tmpfd = -1;
 
-    if ((!identical) || (!this) || (!this->private)) {
+    if ((!identical) || (!this->private)) {
         gf_smsg(THIS->name, GF_LOG_ERROR, errno, GD_MSG_INVALID_ARGUMENT, NULL);
```

Comment:
nit: this->name

Reply:
Ack..seeing too many this and THIS is not good for eyes I guess :p